pkg/plugin/authorization/tokens.go

```diff
@@ -0,0 +1,48 @@
+package authorization
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/hellofresh/janus/pkg/config"
+	"github.com/hellofresh/janus/pkg/models"
+)
+
+func UpdateTokens(conf *config.Config, tokenManager *models.TokenManager) error {
+	url := fmt.Sprintf("%s/%s/tokens", conf.UserManagementURL, conf.ApiVersion)
+
+	resp, err := http.DefaultClient.Get(url)
+	if err != nil {
+		return err
+	}
+
+	defer resp.Body.Close()
+
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return err
+	}
+
+	//newTokens := []*models.JWTToken{}
```

Comment:
If this is unnecessary code, remove it. Otherwise, please add a TODO

Reply:
Done